metrics/github/backfill.py

```diff
@@ -100,39 +198,32 @@ def open_on_day(pr, start, end):
 
 
 def pr_throughput(prs, org):
-    start = date_from_iso(min([pr["created"] for pr in prs]))
+    start = min([pr["created"] for pr in prs])
     days = list(iter_days(start, date.today()))
 
     with TimescaleDBWriter(GitHubPullRequests) as writer:
         for day in days:
-            opened_prs = [pr for pr in prs if date_from_iso(pr["created"]) == day]
+            opened_prs = [pr for pr in prs if pr["created"] == day]
             log.info("%s | %s | Processing %s opened PRs", day, org, len(opened_prs))
             process_prs(writer, opened_prs, day, name="prs_opened")
 
-            merged_prs = [
-                pr for pr in prs if pr["merged"] and date_from_iso(pr["merged"]) == day
-            ]
+            merged_prs = [pr for pr in prs if pr["merged"] and pr["merged"] == day]
+            merged_prs = drop_archived_prs(merged_prs, key="merged")
             log.info("%s | %s | Processing %s merged PRs", day, org, len(merged_prs))
             process_prs(writer, merged_prs, day, name="prs_merged")
 
 
 @click.command()
 @click.argument("org")
-@click.option("--pull-data", is_flag=True, default=False)
-@click.option("--db-path", type=str, default="github.db")
 @click.pass_context
-def backfill(ctx, org, pull_data, db_path):
+def backfill(ctx, org):
     """Backfill GitHub data for the given GitHub ORG"""
-    if pull_data:
-        # clean up existing db
-        Path(db_path).unlink(missing_ok=True)
-
-        # pull all data down to make backfilling quicker
-        get_data(db_path, org)
-
-    prs = get_prs(db_path)
+    prs = list(iter_prs(org))
 
     org_prs = [pr for pr in prs if pr["org"] == org]
```

Comment:
I don't understand this filter. You're only dealing with PRs for a single org, aren't you?

Reply:
Oops, thanks for the catch 👍  This is a vestige of this module starting life as a generic script 🤦‍♂️

Reply:
Fixed in 1f9225b